generator/templates/model.blank.tpl

```diff
@@ -0,0 +1,20 @@
+<?php
+/*
+ * Copyright 2014 Google Inc.
```

Comment:
Ideally the copyright year should be the year that the file is first added. It's difficult to do that in generator. Maybe defer to the synth script?

Should we use Google LLC instead?

True for other files